src/scripts/components/foundations/Button.js

```diff
@@ -95,27 +50,54 @@ export const ButtonStyleHover = css`
   }
 `
 
-export const ButtonColor = styled.button`
-  ${ButtonStyle} ${ButtonStyleHover} ${ButtonStyleBackground} border-radius: 2px;
-  color: ${props => props.theme.colors.button.white};
-  padding: 8px 18px;
+export const ButtonStyleBackground = css`
```

Comment:
Since these styles do seem to be shared, can we put them in a separate file/directory? I do not like exporting styles from components directly; just seems like an atypical pattern